asciidoctor_installation.adoc

```diff
@@ -0,0 +1,137 @@
+= Introduction to Asciidoctor
+
+Asciidoctor is a fast text processor and publishing toolchain for converting AsciiDoc content to HTML5, DocBook and other formats.
+It is written in Ruby and packaged as a Ruby gem.
+
+== Installing Asciidoctor
+
+.Prerequisites
+* Asciidoctor requires one of the following implementations of Ruby:
+** Ruby version 2.3 or later
+** JRuby version 9.1 or later
+** TruffleRuby version 20.2 or later
+* Supported operating systems:
+** Ubuntu versions 18.04 LTS and 20.04 LTS
+** Fedora version 33 or later
+** macOS Mojave, Catalina, and Big Sur
+** Microsoft Windows 10 and Windows Server 2019
+
+.Procedure
+Use your operating system package manager to install Asciidoctor:
+
+* xref:linux[Install on Linux]
+* xref:macos[Install on macOS]
+* xref:windows[Install Windows]
+
+[[linux]]
+.Linux operating system
+. Use Linux package manager to install Asciidoctor:
+.. On Debian-based distributions such as Ubuntu, use APT to install Asciidoctor.
++
+[source,bash]
+----
+sudo apt-get install -y asciidoctor
+----
+
+.. On RPM-based Linux distributions, such as Fedora, CentOS, and RHEL, use the DNF to install Asciidoctor.
++
+[source,bash]
+----
+sudo dnf install -y asciidoctor
+----
++
+. Optional: Visit link:https://pkgs.org/search/?q=asciidoctor-pdf[https://pkgs.org/search/?q=asciidoctor-pdf] to install package for converting AsciiDoc documents to PDF.
```

Comment:
Why optional? This should be a requirement, too. Also, the instruction is not to go to pkgs.org but use the standard way of installing packages, like apt-get, dnf etc. I just mentioned pkgs.org to help you figure out the package names in different distros...

Reply:
Hey Petr
Optional because its not required? But no problem I changed it
I managed to find the package names and updated the procedure